book/doctrine.rst

```diff
@@ -951,8 +951,6 @@ To relate the ``Category`` and ``Product`` entities, start by creating a
                 products:
                     targetEntity: Product
                     mappedBy: category
-            # don't forget to init the collection in the __construct() method
```

Comment:
OK, then please change it to:
# The ArrayCollection 'products' is automatically initialized in the __construct() method, when you call doctrine:generate:entities